index.html

```diff
@@ -31,7 +31,7 @@
         action="https://www.freecodecamp.com/email-submit"
         method="POST"
       >
-        <h1>Check your account</h1>
+        <h1>Check your  account</h1>
```

Comment:
To make your code more professional  try always to not use two `h1` or `h2`, it would be better to avoid  2 headlines from the same level

Reply:
- [ ] Just as I mentioned before, I will suggest that you make use of polite language when giving reviews so it doesn't sound like you're giving orders to the student :+1: 
- These are polite suggestions: `"I think it would be better to use semantic tags", I will suggest...., Kindly consider...... etc`
- Kindly ensure that you add the reasons why a particular change is requested. You might know why you're requesting the change but the student might not know why so to avoid confusion, it is always advisable to add "Why" a change is requested :+1: 
- [ OPTIONAL ]  will suggest that you include emojis in your reviews so it sends a friendly tone to the recipient. :100: